kubernetes/ccloudexporter.dp.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ccloudexporter
+  name: ccloudexporter
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
```

Comment:
That's the default values, is there a strong requirement to set it up?
Otherwise, I would remove those to avoid confusion.

Reply:
Yes you can proceed. I did a `kubectl get deployment... > ccloudexporter.dp.yaml `and did not remove all included lines :)